extensions/ql-vscode/package.json

```diff
@@ -1004,6 +1012,10 @@
           "command": "codeQLDatabasesExperimental.setSelectedItem",
           "when": "false"
         },
+        {
+          "command": "codeQLDatabasesExperimental.setSelectedItemContextMenu",
+          "when": "false"
```

Comment:
When does this evaluate? I can see the context action if 'false' is set and also when the whole line is removed..